scene/gui/slider.cpp

```diff
@@ -163,6 +191,39 @@ void Slider::_update_theme_item_cache() {
 
 void Slider::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_INTERNAL_PROCESS: {
+			Input *input = Input::get_singleton();
+
+			if (input->is_action_just_released("ui_left") || input->is_action_just_released("ui_right") || input->is_action_just_released("ui_up") || input->is_action_just_released("ui_down")) {
```

Comment:
Same problem as in https://github.com/godotengine/godot/pull/63168#discussion_r1170564401, but I guess it's a bit more complicated to handle. You could leave it like that.